proto/data.proto

```diff
@@ -30,12 +30,13 @@ message DataType {
     MINUTE_TO_SECOND = 13;
   }
   enum TypeName {
-    INT16 = 0;
-    INT32 = 1;
-    INT64 = 2;
-    FLOAT = 3;
-    DOUBLE = 4;
-    BOOLEAN = 5;
+    TYPE_UNSPECIFIED = 0;
```

Comment:
Ditto.

Reply:
We can't remove it, because we defined two same name in one message